ddl/ddl.go

```diff
@@ -61,7 +62,9 @@ type DDL interface {
 	// all servers have the same lease time.
 	SetLease(lease time.Duration)
 	// Stat returns the DDL statistics.
-	Stat() (map[string]interface{}, error)
+	Stat() (map[string]*variable.StatusVal, error)
```

Comment:
you can pass scope as the Stat argument and let Statist itself determine which status to return.
and I think GetDefaultStatusScopes is not necessary.